src/kit/taosdemo/taosdemo.c

```diff
@@ -2735,15 +2738,15 @@ static int printfInsertMeta() {
                     g_Dbs.db[i].superTbls[j].sampleFile);
             printf("      tagsFile:          \033[33m%s\033[0m\n",
                     g_Dbs.db[i].superTbls[j].tagsFile);
-            printf("      columnCount:       \033[33m%d\033[0m\n",
+            printf("      columnCount:       \033[33m%d\033[0m\n        ",
                     g_Dbs.db[i].superTbls[j].columnCount);
             for (int k = 0; k < g_Dbs.db[i].superTbls[j].columnCount; k++) {
                 //printf("dataType:%s, dataLen:%d\t", g_Dbs.db[i].superTbls[j].columns[k].dataType, g_Dbs.db[i].superTbls[j].columns[k].dataLen);
                 if ((0 == strncasecmp(g_Dbs.db[i].superTbls[j].columns[k].dataType,
                                 "binary", 6))
                         || (0 == strncasecmp(g_Dbs.db[i].superTbls[j].columns[k].dataType,
                                 "nchar", 5))) {
-                    printf("column[\033[33m%d\033[0m]:\033[33m%s(%d)\033[0m ", k,
+                    printf("column[%d]:\033[33m%s(%d)\033[0m ", k,
```

Comment:
why remove colored output?

Reply:
I saw for the tag we don't have color, so I just keep the column and tag the same.